base/libgit2/libgit2.jl

```diff
@@ -603,8 +605,14 @@ function __init__()
 
     err = ccall((:git_libgit2_init, :libgit2), Cint, ())
     err > 0 || throw(ErrorException("error initializing LibGit2 module"))
+    REFCOUNT[] += 1
```

Comment:
Ideally we'll need to replace these with atomic counters but this should be good enough for now.